Apple/Burrow.xcodeproj/project.pbxproj

```diff
@@ -329,13 +341,21 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D020F66229E4A6E5002790F6 /* NetworkExtension.xcconfig */;
 			buildSettings = {
+				DEVELOPMENT_TEAM = P6PV2R9443;
```

Comment:
These are changes made to the Xcode project that we likely don't want – you can delete these settings in Xcode by going to the Network Extension target in Xcode, to "Build Settings", show "Levels" and "Customized" and you can see which settings you need to delete

In general, we only keep Xcode settings in `xcconfig` files